bin/GenerateCategories.php

```diff
@@ -26,7 +26,7 @@ public function prepare_categories( $file ) {
 		'// This file was generated using GenerateCategories.php, do not modify it manually' . PHP_EOL .
 		'// php GenerateCategories.php taxonomy-with-ids.en-US.txt' . PHP_EOL .
 		'class GoogleProductTaxonomy {' . PHP_EOL .
-		'	public const TAXONOMY = %s' . PHP_EOL .
+		'	const TAXONOMY = %s' . PHP_EOL .
```

Comment:
> Since we required public and public is implicit removing the const keyword is all that is needed

Just to clarify - the fix is to remove `public`, not remove `const` as mentioned in description!

Reply:
Yes, sorry.